projects/fusion-ui/components/input/input.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="fu-input-holder {{ config.options?.size }}"
+<div [class.fu-input-small]="config.options?.size === 'small'" class="fu-input-holder"
```

Comment:
use condition in setter instead
`config.options?.size === 'small'`

Reply:
I've change it to get:
get isSmall(): boolean {
        return this.config.options?.size === 'small';
    }